cmd/kubeadm/app/constants/constants.go

```diff
@@ -312,6 +312,9 @@ const (
 	// KubeDNSKubeDNSImageName specifies the name of the image for the kubedns container in the kube-dns add-on
 	KubeDNSKubeDNSImageName = "k8s-dns-kube-dns"
 
+	// KubeDNSReplicas is the default replica count for kube-dns DNS deployments
+	KubeDNSReplicas = 1
```

Comment:
I don't think, that these consts are used outside of `dns.go`, so we can move them there and even make them package private.

Reply:
Moved into `dns.go`